### PR TITLE
test: add adapters test

### PR DESCRIPTION
**Pull request by zanelladev**

### CODE REVIEW

File: test/src/modules/cache/external/adapters/cache_adapter_test.dart [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import 'package:auto_cache_manager/src/modules/cache/domain/entities/cache_entity.dart';
+import 'package:auto_cache_manager/src/modules/cache/domain/enums/invalidation_type.dart';
+import 'package:auto_cache_manager/src/modules/cache/domain/enums/storage_type.dart';
+import 'package:auto_cache_manager/src/modules/cache/external/adapters/cache_adapter.dart';
+import 'package:auto_cache_manager/src/modules/cache/external/adapters/enums/invalidation_type_adapter.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('CacheAdapter.fromJson |', () {
+    test('should be able to get CacheEntity from json successfully', () async {
```
Comment (luizgasparetto) [inline test/src/modules/cache/external/adapters/cache_adapter_test.dart:10]: Retirar ASYNC

File: test/src/modules/cache/external/adapters/cache_adapter_test.dart [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      // Arrange
+      const id = 'zanelladev';
+      const data = 'test';
+      const storageType = StorageType.kvs;
```
Comment (luizgasparetto) [inline test/src/modules/cache/external/adapters/cache_adapter_test.dart:14]: StorageType n sendo usado

File: test/src/modules/cache/external/adapters/cache_adapter_test.dart [hunk 3]
```diff
@@ -0,0 +1,69 @@
+import 'package:auto_cache_manager/src/modules/cache/domain/entities/cache_entity.dart';
+import 'package:auto_cache_manager/src/modules/cache/domain/enums/invalidation_type.dart';
+import 'package:auto_cache_manager/src/modules/cache/domain/enums/storage_type.dart';
+import 'package:auto_cache_manager/src/modules/cache/external/adapters/cache_adapter.dart';
+import 'package:auto_cache_manager/src/modules/cache/external/adapters/enums/invalidation_type_adapter.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('CacheAdapter.fromJson |', () {
```
Comment (luizgasparetto) [inline test/src/modules/cache/external/adapters/cache_adapter_test.dart:9]: Faltou teste de erro

File: test/src/modules/cache/external/adapters/cache_adapter_test.dart [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+      expect(cache.id, equals(id));
+      expect(cache.data, equals(data));
+      expect(cache.storageType, equals(storageType));
+      expect(cache.invalidationType, equals(invalidationType));
+      expect(cache.createdAt, equals(createAt));
+    });
+  });
+
+  group('CacheAdapter.toJson |', () {
+    test('should be able to parse CacheEntity to json successfully', () async {
```
Comment (luizgasparetto) [inline test/src/modules/cache/external/adapters/cache_adapter_test.dart:42]: Tirar ASYNC

File: test/src/modules/cache/external/adapters/cache_adapter_test.dart [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+        storageType: storageType,
+        invalidationType: invalidationType,
+        createdAt: createAt,
+      );
+
+      // Act
+      final jsonCache = CacheAdapter.toJson(cache);
+
+      // Assert
+      expect(jsonCache['id'], equals(id));
```
Comment (luizgasparetto) [inline test/src/modules/cache/external/adapters/cache_adapter_test.dart:62]: Add .containsKey

**Review by luizgasparetto — COMMENTED** [review]
KLKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKK

**Comment by zanelladev** [comment]
> KLKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKK

vsfddd kkkkk

**Review by luizgasparetto — APPROVED** [review]
LGTM :)

**Pull request merged**